### PR TITLE
Add mixin for creating text-ellipses

### PR DESCRIPTION
**Pull request by nicomelamed**
Description of the use:
If you only need text ellipsis in only one line you can use: 
`@include text-ellipsis` and it works with one line, and this is the result:

text-ellipsis1

In case that you need text ellipsis in the third line of text you can pass the number of lines as a parameter:
`@include text-ellipsis(3)` and this is the result:

text-ellipsis

### CODE REVIEW

**Review by pigeonfresh — APPROVED** [review]

File: utils/mixin/_text-ellipsis.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * @param {number} $lines - Numbers of lines to make the text overflow ellipsis
+ */
+@mixin text-ellipsis($lines: 1) {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	@if $lines > 1 {
+		display: -webkit-box;
+		-webkit-line-clamp: $lines;
+		-webkit-box-orient: vertical;
+		visibility: visible;
```
Comment (larsvanbraam) [inline utils/mixin/_text-ellipsis.scss:13]: Why do we also set the visibility?

**Review by ThaNarie — CHANGES_REQUESTED** [review]
Can add your PR explanation in the code comments, and also update the readme/docs in this repo? :)

File: utils/mixin/_text-ellipsis.scss (hunk 1 above)
Comment (nswaldman) [reply utils/mixin/_text-ellipsis.scss:13]: Would recommend adding `overflow: hidden;`

Comment (nicomelamed) [reply utils/mixin/_text-ellipsis.scss:13]: The overflow hidden is above in the default parameters, and the visibility visible is an ios issue that I solved with the visibility, I think it's related to gsap and all the inline styles that gsap adds.

**Comment by nicomelamed** [comment]
> Can add your PR explanation in the code comments, and also update the readme/docs in this repo? :)

Done!

**Comment by pigeonfresh** [comment]
@ThaNarie does this meet your request or are additional actions needed?

**Review by ThaNarie — APPROVED** [review]

**Pull request merged**